libs/eventually-broker/src/routes/subscriptions.ts

```diff
@@ -159,10 +159,10 @@ router.get("/:id", async (req, res) => {
           ...props,
           ...sub
         })
-      : res.render("edit-subscription", { ...props, ...err });
+      : res.render("edit-subscription", { ...props, ...err, isAdmin: isAdmin(req)});
   } catch (error) {
     log().error(error);
-    res.render("edit-subscription", { ...props, ...err });
+    res.render("edit-subscription", { ...props, ...err, isAdmin });
```

Comment:
property or function? why not declare a const isAdmin = utils.isAdmin(req) and reuse it?

Reply:
I forgot to update this line, will update it now